nipap-cli/nipap

```diff
@@ -53,13 +53,11 @@ if __name__ == '__main__':
     nullh = NullHandler()
     logger.addHandler(nullh)
 
-    import optparse
-    parser = optparse.OptionParser()
-    parser.add_option("-f", "--force", action="store_true", help="disable interactive prompting of actions")
-    parser.add_option("--show-interpretation", action="store_true", help="show search interpretation")
```

Comment:
I'm guessing there was a conflict here and show interpretation was accidentally deleted.